data/dataset.urls

```diff
@@ -47,3 +49,5 @@ http://crd-esgf-drc.ec.gc.ca/thredds/fileServer/esgC_dataroot/AR6/CMIP6/CMIP/CCC
 http://crd-esgf-drc.ec.gc.ca/thredds/fileServer/esgC_dataroot/AR6/CMIP6/CMIP/CCCma/CanESM5/historical/r2i1p2f1/Amon/tas/gn/v20190429/tas_Amon_CanESM5_historical_r2i1p2f1_gn_185001-201412.nc
 http://crd-esgf-drc.ec.gc.ca/thredds/fileServer/esgC_dataroot/AR6/CMIP6/CMIP/CCCma/CanESM5/historical/r3i1p2f1/Amon/tas/gn/v20190429/tas_Amon_CanESM5_historical_r3i1p2f1_gn_185001-201412.nc
 http://crd-esgf-drc.ec.gc.ca/thredds/fileServer/esgC_dataroot/AR6/CMIP6/CMIP/CCCma/CanESM5/historical/r4i1p2f1/Amon/tas/gn/v20190429/tas_Amon_CanESM5_historical_r4i1p2f1_gn_185001-201412.nc
+# Variable = pr, model = GPCP-SG, project= obs4mips
+https://dpesgf03.nccs.nasa.gov/thredds/fileServer/obs4MIPs/observations/NASA-GSFC/Obs-GPCP/GPCP/V2.2/atmos/pr/pr_GPCP-SG_L3_v2.2_197901-201312.nc
```

Comment:
When running example (see #126), still get missing data error.

Seems `pr_GPCP-SG_L3_v2.2_197901-201312.nc` needs to be in `~/esmvaltool_tutorial/data/Tier1/GPCP-SG`. The wget command in the setup episode puts it in  `~/esmvaltool_tutorial/data/`. 

We need to put it in the right place somewhere.

Reply:
After wget command we could add?

```shell
mkdir -p Tier1/GPCP-SG
mv pr_GPCP-SG_L3_v2.2_197901-201312.nc Tier1/GPCP-SG/
```

Reply:
@sverhoeven thanks. I added these lines.